modules/nf-core/xeniumranger/import-segmentation/main.nf

```diff
@@ -0,0 +1,82 @@
+process XENIUMRANGER_IMPORT_SEGMENTATION {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "nf-core/xeniumranger:3.0.1"
+
+    input:
+    val(meta)
+    path(xenium_bundle)
+    val(expansion_distance)
+    path(coordinate_transform)
+    path(nuclei)
+    path(cells)
+    path(transcript_assignment)
+    path(viz_polygons)
```

Comment:
Sample specific data should go in the same tuple as `meta`.
Reference data should either go in one tuple with it's own `meta` or in separate channels per datatype, again each with it's own `meta`

example: https://github.com/nf-core/modules/blob/master/modules/nf-core/snapaligner/align/main.nf#L10

Reply:
Code Changed -> Tested -> Updated PR ✌🏻